app/controllers/catalog_controller.rb

```diff
@@ -11,28 +11,29 @@ class CatalogController < ApplicationController
 
   def index
     super
-    load_lookup_tables
     $brand = if params.include?('lib')
                params['lib']
              else
                'neos'
              end
-    $libraryname = @libraries[$brand]['name']
-    $homeurl = @libraries[$brand]['url']
-    $neosurl = @libraries[$brand]['neosurl']
+    library = Library.find_by!(short_code: $brand)
```

Comment:
What do we want to happen if the Library is not found?

Reply:
Thinking this through. With the logic as it currently stands, if it doesn't find a specific library, it will default to Library.find_by!(short_code: 'neos'). Probably the safest bet would be to have a NEOS entry in the db table with name, homeurl, and neos url as something appropriate to NEOS as a whole, and with no proxy prefix. If a library is *missing* from the NEOS config, then they will see the lack of branding, etc, and get in touch. Does that make the most sense?